pubpub/pubpub
# bug: display of popper citation

addresses #2003 

It was all in the popper vesion.
I tried wrestling with v. 1.x `3-D-transform` property, but that wasn't going anywhere. So I jumped us up to popperJS 2.something, and, well, it just works.

_Test Plan_

+ visit a pub with heavy-hitting citations, like the [JBT AMA Citation Testing pub](pub/k72aiow0/release/1)
+ find a succulent inline citation with lots of tasty text
+ click that mamma jamma and make sure the citation that's rendered isn't spilling over the viewport
+ repeat with increasingly narrow viewports to your heart's content.
